changelogs/fragments/454-legacy_cleanup.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

```suggestion
trivial:
```

This is a fine change that is long overdue, but I don't think a change to an example in a docstring of an internal class is really rising to the level of a bugfix.